alembic/versions/411bcc6b39b8_adding_property_to_author_table.py

```diff
@@ -0,0 +1,28 @@
+"""Adding @property to author table
+
+Revision ID: 411bcc6b39b8
+Revises: 9e475da39630
+Create Date: 2019-11-05 08:07:03.873493
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '411bcc6b39b8'
+down_revision = '9e475da39630'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
You still have an empty migration. I'll approve but make a subsequent PR to remove it.